index.html

```diff
@@ -51,23 +54,41 @@
                     <!--Cart-->
                 </ul>
             </div>
+            
+        <!-- START OUR POP-UP -->
+            <div class="pop-up-wrapper active" id="pop-up-wrapper">
+                <!--Creates the popup body-->
+                <div class="popup-overlay active-popup">
+                    <!--Creates the popup content-->
+                    <div class="popup-content active-popup">
+                        <div class = "popup-photo">
+                            <img src="img/wine.PNG">
+                        </div>
+                        <div class = "popup-text">
+                            <h1>NEVER MISS A NEW RELEASE</h1>
+                            <br>
+                            <p>We'd love to stay in touch with you and send a postcard from our Planet Winery time to time. Join our list to stay up to data on all things Planet Winery</p>
+                         </div>
+                        <!--popup's close button-->
+                        <div class = closeButton>
+                            <button class="close close-button-layout"><i class="fas fa-times"></i></i></button>
+                        </div>
+                     </div>
+                </div>
+            </div>
 
             <!-- Logo goes here -->
             <a class="desktop-logo" href="index.html"
                 ><img src="img/logo-index.png"
             /></a>
 
             <div class="center-content active-center-content">
-                <!-- Heading Title -->
-                <h1>Planet Wine</h1>
+               <!-- Heading Title -->
+                <h1>Planet Wine</h1> 
```

Comment:
As you have pointed out of this heading, I do think we need to fix this, maybe add issue for this.